rfc/networked-dot-storage.md

```diff
@@ -0,0 +1,174 @@
+
+### Filecoin.Storage
+
+Produce an implementation of W3UP and W3Filecoin that can be run by a Filecoin L1 Storage Provider. In order words, a Filecoin storage provider should be able to be more than a backup on our network -- and eventually get seperate compensation -- if they can meet our service level requirements. See [Filecoin SPs in the W3S Network](https://w3sat.notion.site/Filecoin-SPs-in-the-W3S-Network-16a9b66e7ef3425bbee390c6481bd752?pvs=4)
+
+## FAQ
+
+### Is decentralization the wrong word to use?
+
+Decentralization is both a concept so overloaded it lacks any coherence, and a narrative that our product team has validated is indeed important to many market segments in web3. Actual customers have told us they won't use our product unless it's "decentralized". Likely, in some cases this is a novel kind of CYA (cover-your-ass) -- where enterprise customers want SOC-2 compliance in a data center so they can blame someone else if data gets lost, web3 customers want services they use to be "decentralized" so they can't get called out by the internet for not being "web3". Decentralization is both an annoying meme and an important narrative we have to contend with.
+
+I would say what we're really building is an incentivized permissionless cooperative distributed network for content addressed data. If anyone wants to try to market that, I encourage them to. But internally, maybe we should consider not saying decentralization as much. At minimum, let's align on what we're trying to achieve for our users.
+
+### Does decentralization mean running W3S on a network like Saturn?
+
+Saturn is an interesting experiment in crypto incentization of a retrieval network. There is a lot to learn from that experience. But there is as much to learn about what not to do as what to do. In my opinion, Saturn is over indexed on diversity of small nodes and under indexed on decentralization of coordination. Also, our revenue model with Saturn clearly needs improvement. At minimum money in needs a connection to money out.
+
+### What about quality of service?
+
+Web3.storage users love our reliability, ease of use, and performance. In the content addressed world, these are easily our strongest differentiators. As @alanshaw has pointed out several times, in general, we MUST maintain our level of service as we evolve the network. That is a north star. The world already has enough broken decentralized systems.
+
+### Does "distributed consensus and produce verifiable results" mean smart contracts on a blockchain?
+
+Most likely, yes. But I'm open to hearing about other consensus mechanisms. That said, for the specific problem of trustless consensus on limited execution of relatively small rule sets, honestly, I think blockchains are the most proven solution to date. We need to assess objectively the risks/benefits of a different approach.
```

Comment:
I think it might be worth teasing out what do we need consensus on ? Perhaps some things do not need consensus but could instead lean into giving user a choice ? I think it is not a stretch to describe  blockchains as extremely inefficient coordination system with some mutable state. Which makes me wonder if by consensus we mean coordination and if so perhaps what we actually need is to expose (mutable) state with some concurrency controls.